### ISSUE TITLE
Slot spinner not hide

### ISSUE THREAD

**Issue by mits87** [PeachScript/vue-infinite-loading#22]
Hi,

I use Vue-infinite-loading v.2.0.0-rc.1 and I have problem.
When I add new slot:

```
<span slot="spinner">
  <b>Loading...</b>
</span>
```

and conditions will be true:

```
if (this.items.length === total) {
   this.$refs.infiniteLoading.$emit('$InfiniteLoading:complete');
}
```

then my loader won't hide. 
But when I don't use my own slot spinner then all works fine. 
Could you fix it?

Best Regards
Peter

**Comment by PeachScript** [comment]
Thanks for your report! In fact, this problem also exists in `v1.x` and it existed a long time...

I've fixed it in the latest next version `v2.0.0-rc.2`, please upgrade your version and try it :D

**Comment by mits87** [comment]
Thanks, works perfect!
